src/ansys/grantami/bomanalytics/_item_results.py

```diff
@@ -281,24 +284,31 @@ def parse_reference_type(reference_type: str) -> ReferenceType:
 
 
 class ImpactedSubstance(BaseSubstanceReference):
-    """Represents a substance impacted by a legislation. This object includes two categories of
-    attributes:
+    """Represents a substance impacted by a legislation.
+
+    This object includes two categories of attributes:
 
       - The reference to the substance in Granta MI. These attributes are all populated if data for them exists in
         Granta MI.
       - The amount of the substance in the parent item and the threshold above which it is impacted.
 
-    Attributes
+    Parameters
```

Comment:
Is it ok to leave these as `Attributes`? In this context these fields aren't really parameters that a user will specify to create the class, they are attributes available on the class that has already been created.

Reply:
@Andy-Grigg Sorry! Your stuff is so different than most of what I look at. Given that it builds, it's OK! Also, I'm sure the last time I reviewed this doc you explained that "fluent" was an adjective and not a reference to the Ansys product. Forgive my short-term memory!